app/javascript/noMouseDays.js

```diff
@@ -0,0 +1,8 @@
+const date = new Date();
+const dayOfWeek = date.getDay();
+
+// We only want to disable mouse usage
+// on Fridays.
+if (dayOfWeek === 5) {
+  import('no-mouse-days');
```

Comment:
The [dynamic import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) can just be imported. There is no data returned so just let the Promise execute.